src/Mailer/Email.php

```diff
@@ -2521,7 +2521,7 @@ protected function _attachFiles($boundary = null)
                 !isset($fileInfo['contentDisposition']) ||
                 $fileInfo['contentDisposition']
             );
-            $part = new FormDataPart(false, $data, false);
+            $part = new FormDataPart(false, $data, false, $this->getHeaderCharset());
```

Comment:
The 1st argument of `FormDataPart::__construct()` is string, you can't pass it `false`.

Reply:
Same for the 3rd argument. Use empty string instead for both.